ru/schemas-info.yml

```diff
@@ -0,0 +1,64 @@
+card:
+  architecture:
+    name: Архитектура
+  attack:
+    name: Атака
+    types:
+      error: Ошибка
+  certificate:
+    name: Сертификат
+  component:
+    name: Компонент
+    types:
+      backend: Бекенд
+      frontend: Фронтенд
+      internal_process: Бекенд
+      monitoring: Мониторинг
+      cloud: Облако
+  defense:
+    name: Защита
+    types:
+      escape: Отговорка
+
+attack:
+  first_player_hand: Рука Алисы
+  first_player_table: Стол Алисы
+  second_player_hand: Рука Бориса
+  hidden_card: Закрытая карта
+  reset: Сброс
```

Comment:
`discard`

Reply:
Niceee, I was trying to firgure out what's the best translation for that 😆